scripts/gen_quick_start_module.py

```diff
@@ -123,6 +124,12 @@ def update_versions(versions, release_matrix, release_version):
                     if (x["package_type"], x["gpu_arch_type"], x["gpu_arch_version"]) ==
                     (package_type, gpu_arch_type, gpu_arch_version)
                     ]
+                if os_key == OperatingSystem.MACOS.value and package_type == "libtorch":
+                    pkg_arch_matrix += [
+                        x for x in release_matrix[OperatingSystem.MACOS_ARM64.value]
+                        if (x["package_type"], x["gpu_arch_type"], x["gpu_arch_version"]) ==
+                        (package_type, gpu_arch_type, gpu_arch_version)
+                        ]
```

Comment:
Can you please add some sort of a comment to explain why those complications are necessary?

Reply:
Ok Will do. Basically we are trying to reduce 2 Matrices MACOS and MACOS_ARM64 to 1 MacOS matrix that we display to user. This is only applicable to MacOS and currently only to libtorch.